sagemaker-core/spark-processing-sm-core.ipynb

```diff
@@ -0,0 +1,1059 @@
+{
```

Comment:
<p>No need to set region here, we can let users to use their default region</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4748/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Done